spatialos-sdk/src/worker/connection.rs

```diff
@@ -424,8 +425,10 @@ impl Connection for WorkerConnection {
             reserved: ptr::null_mut(),
             component_id: C::ID,
             command_index,
-            schema_type: ptr::null_mut(),
-            user_handle: vtable::handle_allocate(request),
+            // TODO: Do we need to free the `Schema_CommandRequest` object at this point? Or will
+            // the SDK handle that for us?
```

Comment:
I believe the SDK handles that for you

Reply:
Excellent, that's what I had hoped! I've removed those comments.